tide_api.install

```diff
@@ -20,3 +20,25 @@ function tide_api_install() {
   Role::load('anonymous')->grantPermission('access jsonapi resource list')->save();
   Role::load('authenticated')->grantPermission('access jsonapi resource list')->save();
 }
+
+/**
+ * Implements hook_update_dependencies().
+ */
+function tide_api_update_dependencies() {
+  $dependencies['tide_api'][8001] = [
```

Comment:
Does this make sense? According to [this link](hook_update_dependencies), this hook is saying that `tide_api_update_8001` must run after `jsonapi_update_8701` but then the next block is saying that `tide_api_update_8001` must run before `jsonapi_update_8701`

Also, the second form is not recommended. From the above link:

> // Indicate that the mymodule_update_8002() function provided by this module
  // must run before the yet_another_module_update_8005() function provided by
  // the 'yet_another_module' module. (Note that declaring dependencies in this
  // direction should be done only in rare situations, since it can lead to the
  // following problem: If a site has already run the yet_another_module
  // module's database updates before it updates its codebase to pick up the
  // newest mymodule code, then the dependency declared here will be ignored.)

Reply:
yes @anthony-malkoun  you are right. my fault. thanks